app/uk/gov/hmrc/helptosavestub/controllers/NSIGetAccountBehaviour.scala

```diff
@@ -0,0 +1,207 @@
+                                         countryCode:               String,
+                                         emailAddress:              String,
+                                         commsPreference:           String,
+                                         clientBlockingCode:        String,
+                                         clientBlockingReasonCode:  String,
+                                         clientCancellationStatus:  String,
+                                         nbaAccountNumber:          String,
+                                         nbaPayee:                  String,
+                                         nbaRollNumber:             String,
+                                         nbaSortCode:               String,
+                                         terms:                     List[Term])
+
+  case class CurrentInvestmentMonth(investmentRemaining: String, investmentLimit: String, endDate: LocalDate)
+
+  object CurrentInvestmentMonth {
+
+    implicit val format: Format[CurrentInvestmentMonth] = Json.format[CurrentInvestmentMonth]
+  }
+
+  case class Term(termNumber: Int, startDate: LocalDate, endDate: LocalDate, maxBalance: String, bonusEstimate: String, bonusPaid: String)
+
+  object Term {
+
+    implicit val format: Format[Term] = Json.format[Term]
+  }
+
+  object NSIGetAccountByNinoResponse {
+
+    implicit val format: Format[NSIGetAccountByNinoResponse] = Jsonx.formatCaseClass[NSIGetAccountByNinoResponse]
+
+    val correlationId: UUID = UUID.randomUUID()
+
+    val bethCIM: CurrentInvestmentMonth = CurrentInvestmentMonth("0.00", "50.00", LocalDate.of(2018, 3, 31))
+
+    val bethTerms: List[Term] = List[Term](Term(1, LocalDate.of(2017, 11, 1), LocalDate.of(2019, 11, 1), "200.00", "100.00", "0.00"),
+                                           Term(2, LocalDate.of(2019, 11, 1), LocalDate.of(2021, 11, 1), "0.00", "0.00", "0.00"))
+
+    def bethNSIResponse(correlationId: Option[String]): NSIGetAccountByNinoResponse = NSIGetAccountByNinoResponse("V1.0", correlationId,
+      1100000112057l, "175.00", "200.00", false, "00", "00", bethCIM, "Beth", "Planner", LocalDate.of(1963, 11, 1), "Line 1", "Line 2",
+      " ", " ", " ", "SV1 1QA", "GB", "email.address@domain.com",
+      "02", "00", "00", " ", "11111111", "Mrs B Planner", " ", "801497", bethTerms)
+
+    val peteCIM: CurrentInvestmentMonth = CurrentInvestmentMonth("9.88", "50.00", LocalDate.of(2018, 3, 31))
+
+    val peteTerms: List[Term] = List[Term](Term(1, LocalDate.of(2017, 9, 1), LocalDate.of(2019, 9, 1), "190.12", "95.06", "0.00"),
+                                           Term(2, LocalDate.of(2019, 9, 1), LocalDate.of(2021, 9, 1), "0.00", "0.00", "0.00"))
+
+    def peteNSIResponse(correlationId: Option[String]): NSIGetAccountByNinoResponse = NSIGetAccountByNinoResponse("V1.0", correlationId,
+      1100000112057l, "165.12", "190.12", false, "00", "00", peteCIM, "Pete", "Loveday", LocalDate.of(1963, 11, 1), "Line 1", "Line 2",
```

Comment:
The different NINOs should perhaps have different associated account numbers to be more realistic.

Reply:
ok, updated.